doc/release/RELEASE-NOTES.md

```
@@ -18,6 +18,26 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
     * `reindex_database`
 
 
+## Release 2.15.1
+
+## Introduction
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)